packages/flutter_hooks/lib/src/primitives.dart

```diff
@@ -4,19 +4,20 @@ part of 'hooks.dart';
 ///
 /// It is typically created by [useRef].
 class ObjectRef<T> {
-  /// A mutable property that will be preserved accross rebuilds.
+  ObjectRef(this.value);
+  /// A mutable property that will be preserved across rebuilds.
   ///
   /// Updating this property will not cause widgets to rebuild.
-  T? value;
+  T value;
 }
 
 /// Creates an object that contains a single mutable property.
 ///
 /// Mutating the object's property has no effect.
 /// This is useful for sharing state accross `build` calls, without causing
 /// unnecessary rebuilds.
-ObjectRef<T> useRef<T>() {
-  return useMemoized(() => ObjectRef<T>());
+ObjectRef<T> useRef<T>(T initialValue) {
+  return useMemoized(() => ObjectRef<T>(initialValue));
```

Comment:
Could you add a test for this? Thanks!

Reply:
Sure, [here](https://github.com/rrousselGit/flutter_hooks/pull/244/commits/763085d244da9dbc03070c0080c47c157844051b) you go!